backend/test/v2/integration/pipeline_k8s_native_api_test.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2018-2023 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"testing"
+	"time"
+
+	"github.com/golang/glog"
+	params "github.com/kubeflow/pipelines/backend/api/v2beta1/go_http_client/pipeline_client/pipeline_service"
+	upload_params "github.com/kubeflow/pipelines/backend/api/v2beta1/go_http_client/pipeline_upload_client/pipeline_upload_service"
+	api_server "github.com/kubeflow/pipelines/backend/src/common/client/api_server/v2"
+	"github.com/kubeflow/pipelines/backend/src/common/util"
+	test "github.com/kubeflow/pipelines/backend/test/v2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+// This test suit tests various methods to import pipeline to pipeline system, including
+// - upload v2 pipeline spec JSON file
+// - upload yaml file
+// - upload tarball file
+// - providing YAML file url
+// - providing tarball file url
+type PipelineK8sApiTest struct {
+	suite.Suite
+	namespace            string
+	resourceNamespace    string
+	pipelineClient       *api_server.PipelineClient
+	pipelineUploadClient *api_server.PipelineUploadClient
+}
+
+// Check the namespace have ML job installed and ready
+func (s *PipelineK8sApiTest) SetupTest() {
```

Comment:
As discussed in a meeting, I'd like for us to try and reuse the existing integration tests if possible.